ivy/functional/backends/torch/creation.py

```diff
@@ -66,14 +66,9 @@ def arange(
             return torch.arange(start, stop, step=step, device=device, out=out)
     else:
         dtype = as_native_dtype(default_dtype(dtype))
-        if dtype in [torch.int8, torch.uint8, torch.int16]:
-            return torch.arange(
-                start, stop, step=step, dtype=torch.int64, device=device, out=out
-            ).to(dtype)
-        else:
-            return torch.arange(
-                start, stop, step=step, dtype=dtype, device=device, out=out
-            )
+        return torch.arange(
```

Comment:
are you sure this is okay? It doesn't cause Array API tests to fail? I presume it was there for a reason, but maybe it was a mistake or is no longer needed with the latest `torch` version.